user_data.sh

```diff
@@ -0,0 +1,15 @@
+
+##############
+# Install deps
+##############
+
+apt-get -y install python-pip
+
+# Install AWS Client
+pip install --upgrade awscli
+
+aws s3 cp s3://${s3_path} /tmp/user_data.sh
+
+eval "$(cat /tmp/user_data.sh)"
```

Comment:
Why `eval` as opposed to just running it? .... `eval` limits user data to `bash`.

Reply:
because user data does not contains 
'#/bin/bash' to specifiy shell